syntax = "proto3";

package lmjwtest;

// service, encode a plain text 
service PredictService {
    // request a service of encode
    rpc Predict(input) returns (response) {}
}


message input {
    string inputType = 1;
    string inputStream = 2;
}

message response {
    string status = 1;
}
